graylog2-web-interface/src/stores/configurations/ConfigurationsStore.js

```diff
@@ -2,13 +2,14 @@
 import Reflux from 'reflux';
 import { qualifyUrl } from 'util/URLUtils';
 import fetch from 'logic/rest/FetchProvider';
+import ApiRoutes from 'routing/ApiRoutes';
 import UserNotification from 'util/UserNotification';
 
 import ActionsProvider from 'injection/ActionsProvider';
 
 const ConfigurationActions = ActionsProvider.getActions('Configuration');
 
-const urlPrefix = '/system/cluster_config';
+const urlPrefix = ApiRoutes.ClusterConfigResource.config().url;
```

Comment:
Not sure if these changes are relevant to the Customization Plugin system?

Reply:
Not for the plugin system but for the header badge which is part of the customization page.